### ISSUE TITLE
No bridges Found

### ISSUE THREAD

**Issue by beregor** [elgatosf/streamdeck-philipshue#10]
Plugin been working for ages but today it refuse to connect and it cant find the bridge.
restarted the philips hub same issue
i can control the lights from the computer using the philips hue software so its not connection issue
tryed to start streamdeck as admin and no admin still same issue.

im running out of ideas.
is there anyway to connect via ip adress?

**Comment by proactivematter** [comment]
This just started happening to me as well. Coincidence? Pulling my hair out here because I was acctually copying my profiles from my MAC to use in my new windows system when I noticed I couldn't control / find / pair the with the bridge anymore from the Stream Deck.

**Comment by SoWhoYou** [comment]
Just started happening to me today as well, I even messaged the original developer of the plugin on discord, no fix yet :(

**Comment by SoWhoYou** [comment]
![HuePlugin](https://user-images.githubusercontent.com/5152979/107720992-87e2e980-6ca9-11eb-84af-1f9e07542b46.gif)

**Comment by proactivematter** [comment]
Oh wow, am I glad to have found you guys then. @SoWhoYou thanks for reporting in and going through the motions so the rest of us don't have to. I guess this is widespread and we just have to wait for a a fix then.

**Comment by proactivematter** [comment]
Well, for whatever reason it can now find my bridge and says pairing is successful, but afterwards the bridhe will not show up for selection in the stream deck actions.

**Comment by SoWhoYou** [comment]
Issue now spread to reddit as well, I've also emailed elgato and submitted a support ticket about this issue.

Sadly controlling my Hue Lights and Switches is one of the last things I still use my streamdeck for.
So I'd like to have the functionality back that I had a few hours ago prior to this issue starting earlier today.

https://www.reddit.com/r/elgato/comments/lhz66a/streamdeck_hue_integration_no_longer_working/?utm_medium=android_app&utm_source=share

**Comment by proactivematter** [comment]
Well, whatever it was, at least for me it appears to be solved. Working now on both my Mac and WIndows machines. Are you guys still having issues?

**Comment by SoWhoYou** [comment]
Well it's working again, not sure if the reports made caused someone to fix something, or if HUE was just updating things on their end and caused issues or what not. But whatever I guess, all seems to be working again now. Now I can spend time resetting up my buttons. Along with fixing another HUE issue I'm having were devices in another room are also being controlled via actions in a different room that shouldn't be.... meh it's been a long day. Lets just hope the streamdeck hue controls stay fixed lol.

**Comment by proactivematter** [comment]
@SoWhoYou Your buttons reset? All my hue buttons restored their scenes when the bridge paired. Even transfering my profiles from Mac to Windows I was pleasantly surprised that all Hue buttons (all buttons actually) retained their information and no further re-configuring was required.

**Comment by SoWhoYou** [comment]
@proactivematter I actually deleted them all myself. I did a full stream deck reinstall and many other things like factory resetting my hub and stuff like that trying to get things working again prior to this issue was reported. Sadly I didn't back things up, then I had issues with the echo in the living room controlling things in the bedroom and the other way around as well. Needless to say, my day has just been an automated hell with multiple rooms and it all started with the streamdeck plugin. Had the plugin not stopped working, I wouldn't have reset things and I wouldn't have had to recreate routines in different rooms. Now I've gotten all my routines recreated so all that's really left is to recreate the 50+ streamdeck hot keys for all the devices. I just have an issue with automation, the issue being that I love to automate things. I love doing it so much that I even have hot keys on my streamdeck for over locking my gpus, underclocking them, charging lipo batteries and so much more, it's insane really. I could've easily saved myself this headache if I had just walked away or ignored the issue once I noticed it, but me being me... yeah I can't ignore things unfortunately, I just have to mess with things. Regardless I guess things will be a little more organized now though since I'm basically redoing everything from scratch, including hue lighting scenes which is the most annoying tbh. Sorry for the long rant I like typing, especially when on mobile lol.
